### ISSUE TITLE
After upgrading to Vagrant 2.0 we get "No host IP was given to the Vagrant core NFS helper"

### ISSUE THREAD

**Issue by MikaelWahlberg** [nsidc/vagrant-vsphere#250]
Only way I found to fix it was to disable NFS with: 
    override.nfs.functional = false

Any other ideas, or is it a bug?   It did work with 1.9.x without this (Note, we are not using NFS for anything, so it isn't a problem disabling it for now)

**Comment by AlPhi247** [comment]
I am facing the same problem: 
Works with Vagrant 2.0.1. 
Fails with Vagrant 2.1.5 and 2.2.4

Did you ever find a solution other than the workaround you've already mentioned?

**Comment by MikaelWahlberg** [comment]
No, we just kept the disable of NFS since we don't use it.   So never looked in to this more.

**Comment by AlPhi247** [comment]
Thanks for the quick response. I thought so.
As I don't need the NFS functionality either it is probably not worth wasting my time on this..

**Comment by chris2kus31** [comment]
> I am facing the same problem:
> Works with Vagrant 2.0.1.
> Fails with Vagrant 2.1.5 and 2.2.4
> 
> Did you ever find a solution other than the workaround you've already mentioned?

Would anyone know how to properly downgrade to vagrant 2.0.2

**Comment by KaliszAd** [comment]
is this relevant?
```
/usr/share/rubygems-integration/all/gems/vagrant-2.2.7/bin/vagrant:200: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
(eval):1: warning: The called method `error' is defined here
No host IP was given to the Vagrant core NFS helper. This is
an internal error that should be reported as a bug.
```
